infra/aws/terraform/prow-build-cluster/variables.tf

```diff
@@ -71,111 +74,133 @@ variable "vpc_intra_subnet" {
 variable "cluster_name" {
   type        = string
   description = "Name of the EKS cluster"
+  nullable    = false
 }
 
 variable "cluster_region" {
   type        = string
   description = "AWS region of the EKS cluster"
+  nullable    = false
 }
 
 variable "cluster_version" {
   type        = string
   description = "Kubernetes version of the EKS control plane"
+  nullable    = false
 }
 
 variable "node_group_version_us_east_2a" {
   type        = string
   description = "Kubernetes version of the EKS-managed node group in US East 2a"
+  nullable    = false
 }
 
 variable "node_group_version_us_east_2b" {
```

Comment:
We could refactor this to a map by AZ, instead.

Reply:
We will drop all `node_group_*` fields in the coming days.